Support/Scripts/ds2-gdb-wrapper.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
```

Comment:
Nit: I think having a space between the `!` and the `/bin/bash` is non-standard.